client/src/includes/initSkipLink.test.js

```diff
@@ -0,0 +1,26 @@
+import initSkipLink from './initSkipLink';
+
+describe('initSkipLink', () => {
+  document.body.innerHTML = `
+  <div>
+  <a id="test" class="skiplink button" href="#main" data-skiplink="">Skip to main content</a>
+  </div>
+  <main id="main" >
```

Comment:
Should be ok for the test but just a reminder to keep an eye out for malformed DOM when writing tests like this.
```suggestion
  <main id="main">
```